src/sql/tables.sql

```diff
@@ -72,6 +72,10 @@ CREATE TABLE msgs (
     timestamp_sent INTEGER DEFAULT 0,
     timestamp_rcvd INTEGER DEFAULT 0,
     hidden INTEGER DEFAULT 0,
+    -- mime_headers column actually contains BLOBs, i.e. it may
+    -- contain non-UTF8 MIME messages.  TEXT was a bad choice, but
+    -- thanks to SQLite 3 being dynamically typed, there is no need to
+    -- change column type.
     mime_headers TEXT,
```

Comment:
Can we not alter the table to change it to `BLOB`? Or is it one of those cases where you need to create a new table, copy over all rows into the new table, then delete the old one?

Reply:
Yes, `ALTER TABLE` still does not support this kind of operation: https://www.sqlite.org/lang_altertable.html
And it does not really matter, `BLOB` and `TEXT` *columns* are not treated different by SQLite except when you try to store integers there and they are coerced to strings for `TEXT` but not `BLOB`: https://www.sqlite.org/datatype3.html
Also even if we could change the column type, we can't change types of values already stored except by fetching and storing them back, it's really not worth going through the whole messages table for that when we can use `TEXT` values already stored just fine.